### ISSUE TITLE
When "Stereo Mix" is used as input device, the sound transferred by mumble 1.4.230 is broken ( mumble 1.3.4 is OK)

### ISSUE THREAD

**Issue by huapage** [mumble-voip/mumble#5568]
### Description

1. Tested under Windows 10 Home 64 bit, Version 20H2
2. Tested with Mumble 1.4.230, 64 bit version
3. When "Stereo Mix" is used as input device, the sound transferred by mumble 1.4.230 is broken ( mumble 1.3.4 is OK)

### Steps to reproduce

1. Open sound settings ( From UI or just run "mmsys.cpl" ), if you cannot find "Stereo Mix" device in "Recording" tab, can try to "Show Disabled Device"
![image](https://user-images.githubusercontent.com/8023738/153784002-8e2ed0df-9b9c-419a-b534-3851dc530394.png)

If you still cannot see "Stereo Mix", then you don't need to do the following test.

2. Enable "Stereo Mix" if needed

3. In Mumble, select "Stereo Mix" as input device
4. Play music with some player ( e.g. VLC player ).
5. Use Mumble in another device to check the music played (Can use mumble for mobile device or mumble for PC if you have a second computer).
6. The test result is for Mumble 1.4.230, it cannot stream the music well (the sound is not continuous), but for Mumble 1.3.4 it can stream the music well. Looks like it is the regression of the new version.

Sample test in version 1.3.4:
![image](https://user-images.githubusercontent.com/8023738/153784320-37928d3a-8d20-4edd-98f3-59f14db60168.png)

Sample test in version 1.4.230
![image](https://user-images.githubusercontent.com/8023738/153784438-7982d2ca-35fb-4dbd-834d-1279b2651c23.png)

### Mumble component

Client

### OS

Windows

### Reproducible?

Yes

### Additional information

If you installed some Virtual Audio Cable and use the Virtual Audio Cable as the input of Mumble, you may have similar test result.

**Comment by huapage** [comment]
Considering the regression in the new version, can we add the link to download old version in https://www.mumble.info/downloads/ ? Currently we need to use https://github.com/mumble-voip/mumble/releases/tag/1.3.4 to download the old version, better provide a link in https://www.mumble.info/downloads (e.g. Download old versions ).

**Comment by Krzmbrzl** [comment]
Do other audio sources work as expected?

Do you have echo cancellation and/or noise cancellation enabled (might have happened by default with the upgrade)?

**Comment by huapage** [comment]
Thanks for the prompt response.

The Microphone audio source seems can work. 

In mumble 1.4.230, the default is set to "Echo Cancellation: Mixed echo cancellation (speex)" and "Noise suppression: RNNoise"

This setting cannot work well for the "Stereo Mix" source.

See the following not working setting:
![image](https://user-images.githubusercontent.com/8023738/153968174-57a4ac03-f39f-487d-883e-6dfcf225dac8.png)
After disabling the Echo Cancellation and set the "Noise suppression to "Speex", the sound is OK now.

The OK setting:
![image](https://user-images.githubusercontent.com/8023738/153969343-1c8531fc-af91-4603-a6e3-a3e154f65b7e.png)

In mumble 1.3.4, the default is set to "Echo Cancellation: Disabled" and "RNNoise" is not select. It can work for "Stereo Mix" source.

Mumble 1.3.4 default setting:
![image](https://user-images.githubusercontent.com/8023738/153968976-08102739-fc72-4d1f-af8e-d94828266424.png)

Do you know why mumble 1.4.230 changed the default value?

**Comment by Krzmbrzl** [comment]
> Do you know why mumble 1.4.230 changed the default value?

I do indeed, since I was the one who changed them (I think) ^^
The reason for enabling echo cancellation by default was that prior to 1.4 the Exil cancellation was kinda broken but it got fixed and after that it seemed that for most use cases, having echo cancellation enabled either improves the experience or at least doesn't degrade it.

The noise suppression also got revamped and since many people reported RNNoise to be miraculously good, we decoded to enable it by default. However, after the release we got several reports about really bad audio quality due to RNNoise, so we will change the default back to Speex in the next release.
